rustfmt.toml

```diff
@@ -1,2 +1,2 @@
 write_mode = "Overwrite"
-
+closure_block_indent_threshold = 1
```

Comment:
Er wait. The diff @philipc posted is if you *don't* have this line, and is an example of why I did add it. Sorry misread the comments.